net/nimble/transport/emspi/src/ble_hci_emspi.c

```diff
@@ -174,6 +174,9 @@ ble_hci_emspi_tx_chunk(const uint8_t *data, int len, int *out_bytes_txed)
     uint8_t buf_size;
     int rc;
 
+    /* Silence spurious "may be used uninitialized" warning. */
+    *out_bytes_txed = 0;
+
```

Comment:
Not important, but you could remove the line which assigns *out_bytes_txd to zero on line 188 now.

Reply:
Thanks - I have included this fix in the "emspi transport - Fix compiler warning." commit.